roles/provision_vcenter/tasks/os_specific_iso_actions/mount_Darwin.yml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
I wouldn't use upper case letters in names. Would be better if you use "{{ var | lower }}" instead.